src/components/EmptyState/EmptyState.stories.js

```diff
@@ -37,22 +44,18 @@ stories.addWithInfo('EmptyState', () => (
       >
         pfExample
       </a>
-    </EmptyState.Help>
-    <EmptyState.Action>
-      <Button bsStyle="primary" bsSize="large" onClick={action('main action')}>
-        Main Action
-      </Button>
-    </EmptyState.Action>
-    <EmptyState.Action secondary>
-      <Button onClick={action('secondary action 1')} title="Perform an action">
-        Secondary Action 1
-      </Button>
-      <Button onClick={action('secondary action 2')} title="Perform an action">
-        Secondary Action 2
-      </Button>
-      <Button onClick={action('secondary action 3')} title="Perform an action">
-        Secondary Action 3
-      </Button>
-    </EmptyState.Action>
-  </EmptyState>
+    </span>
+  ),
+  mainAction: createAction('Main Action'),
+  actions: [
+    'Secondary Action 1',
+    'Secondary Action 2',
+    'Secondary Action 3'
+  ].map(actionName => createAction(actionName))
+});
+
+stories.addWithInfo('EmptyState', () => EmptyStatePattern(getPatternData()));
```

Comment:
i think we can start to use `stories.addWithInfo('EmptyState/Components', () => `

and 

`stories.addWithInfo('EmptyState/Patterns', () => `

to add visual hierarchy to Storybook now. See #268